hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/sender/http/HttpClientsFactory.java

```diff
@@ -44,7 +45,7 @@ public HttpClient createClientForHttp1(String name) {
 
         HttpClient client = new HttpClient(createSslContextFactory());
         client.setMaxConnectionsPerDestination(configFactory.getIntProperty(CONSUMER_HTTP_CLIENT_MAX_CONNECTIONS_PER_DESTINATION));
-        client.setMaxRequestsQueuedPerDestination(configFactory.getIntProperty(CONSUMER_INFLIGHT_SIZE));
+        client.setMaxRequestsQueuedPerDestination(configFactory.getIntProperty(CONSUMER_HTTP_CLIENT_MAX_REQUESTS_QUEUED_PER_DESTINATION));
```

Comment:
From the javadoc: `This parameter controls how many requests can be queued before starting to reject them.`
Do we want to reject messages that cannot be enqueued? How will this affect the `SerialConsumer`?

Reply:
If you don’t have any proxy between hermes-consumers and subscribers then reasonable is to have the same value for CONSUMER_INFLIGHT_SIZE and CONSUMER_HTTP_CLIENT_MAX_REQUESTS_QUEUED_PER_DESTINATION. Otherwise you will probably want to increase CONSUMER_HTTP_CLIENT_MAX_REQUESTS_QUEUED_PER_DESTINATION as for all subscriptions you will have the same address, for example localhost for local proxy.

Answering to your questions:
1. In the first place we don’t want to lead to the situation where http requests queue is full. This is the reason why we would like to increase queue for proxy.
2. If somehow http requests queue will be full then we want to retry sending requests to it and that's how it works now. 
3. Increased value for CONSUMER_HTTP_CLIENT_MAX_REQUESTS_QUEUED_PER_DESTINATION should not affect on SerialConsumer
4. You have added metrics for http requests queue size. Thanks to this we can add a check and detect problematic situations for it.